pkg/config/config.go

```diff
@@ -97,14 +98,32 @@ var (
 	ReadinessServicePath = "/health/readiness"
 )
 
-//CreateClientConfig creates client config
-func CreateClientConfig(kubeconfig string, log logr.Logger) (*rest.Config, error) {
+//CreateClientConfig creates client config and applies rate limit QPS and burst
+func CreateClientConfig(kubeconfig string, qps float64, burst int, log logr.Logger) (*rest.Config, error) {
 	logger := log.WithName("CreateClientConfig")
+
+	clientConfig, err := createClientConfig(kubeconfig, logger)
+	if err != nil {
+		return nil, err
+	}
+
+	clientConfig.Burst = burst
+	if qps > math.MaxFloat32 {
```

Comment:
From an end-user perspective I find it more confusing to essentially convert a float64 to a float32 here. Considering the default is `5.0`  what do you think about erroring out here if `qps > math.MaxFloat32`?

Reply:
That makes sense to me. The value should most likely never be this high anyway.